ambry-api/src/main/java/com.github.ambry/config/FrontendConfig.java

```diff
@@ -152,5 +163,7 @@ public FrontendConfig(VerifiableProperties verifiableProperties) {
     frontendUrlSignerDefaultUrlTtlSecs =
         verifiableProperties.getLongInRange("frontend.url.signer.default.url.ttl.secs", 5 * 60, 0,
             frontendUrlSignerMaxUrlTtlSecs);
+    frontendDefaultRouterGetOption =
+        GetOption.valueOf(verifiableProperties.getString("frontend.default.router.get.option", GetOption.None.name()));
```

Comment:
Minor: could be useful to log an error if the properties had an invalid GetOption string, like missing underscores, rather than silently ignoring.

Reply:
If the string is invalid, the startup will fail actually - this will throw an `IllegalArgumentException` if the string does not match any `GetOption`